test/cli_test.rb

```diff
@@ -0,0 +1,60 @@
+# NOTE: run this test with: $ bundle exec ruby -I lib:test test/cli_test.rb -v
+
+$LOAD_PATH.unshift File.expand_path('../test', __FILE__)
+
+require 'cli_test_helper'
+require 'minitest/autorun'
+require_relative "helpers"
+
+class CLITest < Minitest::Test
+  include Aruba::Api
+  include Helpers
+
+  def setup
+    setup_aruba
+  end
+
+  def test_version
+    version_string = "kbsecret version #{KBSecret::VERSION}."
+    run_command_and_stop "kbsecret version"
+    assert_equal version_string, last_command_started.output.chomp
+  end
+
+  def test_types
+    run_command_and_stop "kbsecret types"
+
+    KBSecret::Record.record_types.each do |type|
+      assert_includes last_command_started.output.chomp, type.to_s
+    end
+  end
+
+  def test_login
```

Comment:
I think it would be good to have one test file per external command, so `kbsecret new` would get `test/cli/test_new.rb` or something similar. The other tests above are for builtins, which probably make sense to put in `test/cli/test_builtins.rb` or `test/cli/test_kbsecret.rb`.

Reply:
(This would also have the benefit of allowing me to merge new test cases incrementally, on a command-by-command basis).

Reply:
These suggestions make sense to me. I can refactor and commit some more tests tonight. 

MiniTest support in Aruba def seems less a priority than Cucumber or RSpec, so it takes a little work to get everything to jibe, and not everything works as well as I might have liked. However, it did seem a good fit with your existing stack & therefore worth proposing. Glad it meets your need.

Reply:
Is ^ that to say you'd prefer the tests on separate commits or pull requests?

Reply:
> MiniTest support in Aruba def seems less a priority than Cucumber or RSpec, so it takes a little work to get everything to jibe, and not everything works as well as I might have liked. However, it did seem a good fit with your existing stack & therefore worth proposing. Glad it meets your need.

If RSpec is easier, I'm definitely open to that! I was actually thinking about converting the library tests to specs, so this would be a good way to kick that off.

> Is ^ that to say you'd prefer the tests on separate commits or pull requests?

Separate commits at least. It's up to you if you'd like to do them in separate PRs (I'd be able to merge them faster that way).